cmd/namespace.go

```diff
@@ -103,9 +108,33 @@ func deleteNamespaceAlias(opts namespaceOptions) error {
 	return nil
 }
 
-func createNamespace(opts namespaceOptions) error {
-	namespaceName := opts.args[0]
+func createNamespaceWithOrgId(opts namespaceOptions, namespaceName, orgId string) error {
+	if !opts.noPrompt {
+		fmt.Printf(`You are creating a namespace called "%s".
+
+This is the only namespace permitted for your organization with id %s.
+
+To change the namespace, you will have to contact CircleCI customer support.
+
+`, namespaceName, orgId)
+	}
+
+	confirm := fmt.Sprintf("Are you sure you wish to create the namespace: `%s`", namespaceName)
+	if opts.noPrompt || opts.tty.askUserToConfirm(confirm) {
+		_, err := api.CreateNamespaceWithOwnerID(opts.cl, namespaceName, orgId)
+
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf("Namespace `%s` created.\n", namespaceName)
+		fmt.Println("Please note that any orbs you publish in this namespace are open orbs and are world-readable.")
+	}
+	return nil
+}
 
+func createNamespaceWithVcsTypeAndOrgName(opts namespaceOptions, namespaceName, vcsType, orgName string) error {
+	var err error
```

Comment:
is this variable needed?

Reply:
Yes, if it is not declared here it will be declared shortly after within the same function. :)

Reply:
I changed it to an implicit assignment!